pkg/issuer/acme/dns/dnsmadeeasy/dnsmadeeasy.go

```diff
@@ -0,0 +1,265 @@
+package dnsmadeeasy
+
+import (
```

Comment:
Was this provider taken from the `xenolf/lego` library? If so, you should ensure correct copyright headers/licenses are included (similar to other DNS providers already in the codebase).

If not, you should ensure that the standard license header is included in this source file 😄